src/views/shared/amount-input/amount-input.view.jsx

```diff
@@ -8,11 +8,11 @@ import { getTransactionFee } from '../../../utils/fees'
 import { TxType } from '@hermeznetwork/hermezjs/src/enums'
 import { getProvider } from '@hermeznetwork/hermezjs/src/providers'
 
-const INPUT_REGEX = /^\d*(?:\.\d*)?$/
```

Comment:
Any idea why it was outside of the function? Is it used anywhere else?

Reply:
no idea, it just used in one function

Reply:
👍